automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -150,6 +154,34 @@ class AuthDomainGroupApiSpec extends AnyFreeSpec with Matchers with WorkspaceFix
       }
 
     }
+
+    "bucket should not be accessible to project owners via projectViewer Google role" in {
+
+      val userA = UserPool.chooseProjectOwner //The project owner who can't see the workspace
+      val userB = UserPool.chooseAuthDomainUser //The user who owns the workspace
+
+      val userAToken: AuthToken = userA.makeAuthToken()
+      val userBToken: AuthToken = userB.makeAuthToken()
+
+      withGroup("AuthDomain", List(userB.email)) { authDomainName =>
+        withCleanBillingProject(userA, List(userA.email), List(userB.email)) { projectName =>
+          withWorkspace(projectName, "AuthDomainGroupApiSpec_workspace", Set(authDomainName)) { workspaceName =>
+
+            val bucketName = Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)(userBToken).parseJson.convertTo[WorkspaceResponse].workspace.bucketName
+
+            eventually {
+              //assert that userA receives 403 when trying to access bucket
+              Google.storage.getBucket(bucketName)(userAToken).status.intValue() should be(403)
+              
+              //assert that userB receives 200 when trying to access bucket (to verify that bucket is set up correctly)
+              Google.storage.getBucket(bucketName)(userBToken).status.intValue() should be(200)
```

Comment:
If the `403` assertion fails, this test will exit immediately, correct?

If so, we should assert for `200` _before_ we assert for `403`. Otherwise, I don't believe we'll cover the scenario David pointed out above.